sympy/physics/mechanics/functions.py

```diff
@@ -162,13 +162,14 @@ def linear_momentum(frame, *body):
     >>> from sympy.physics.mechanics import Point, Particle, ReferenceFrame
     >>> from sympy.physics.mechanics import RigidBody, outer, linear_momentum
     >>> N = ReferenceFrame('N')
+    >>> A_frame = ReferenceFrame('A_frame')
     >>> P = Point('P')
     >>> P.set_vel(N, 10 * N.x)
     >>> Pa = Particle('Pa', P, 1)
     >>> Ac = Point('Ac')
     >>> Ac.set_vel(N, 25 * N.y)
     >>> I = outer(N.x, N.x)
-    >>> A = RigidBody('A', Ac, N, 20, (I, Ac))
+    >>> A = RigidBody('A', Ac, A_frame, 20, (I, Ac))
```

Comment:
This implies that the old way won't work anymore. Why would that be?

Reply:
Otherwise the a center of mass has a velocity in the body-fixed frame.

Reply:
It would now work again. I would however not like to recommend doing so in an example, so I left this change in.